src/ansi-c/c_preprocess.cpp

```diff
@@ -551,6 +551,15 @@ bool c_preprocess_gcc_clang(
 #endif
         argv.push_back("-std=gnu++14");
       break;
+
+    case configt::cppt::cpp_standardt::CPP17:
+#if defined(__OpenBSD__)
```

Comment:
This one is a bit puzzling? Why is OpenBSD different?

Reply:
Yes, that's just copy-pasted from the C++11/14 cases. The change @kroening refers to was made in b8116802b04a85813a38ff69f40df88a9d671ece as part of https://github.com/diffblue/cbmc/pull/4053 and the PR description says that:

> The c_preprocess.cpp source file in ansi-c has been patched to pass the appropriate -std to Clang on OpenBSD. OpenBSD does not use GNU extensions with the system Clang.